### PR TITLE
cliprdr/server: Provide the server with a possibility to configure capabilities

### PR DESCRIPTION
**Pull request by mfleisz**

### CODE REVIEW

**Comment by akallabeth** [comment]
+1

File: include/freerdp/server/cliprdr.h [hunk 1]
```diff
@@ -68,6 +68,12 @@ struct _cliprdr_server_context
 	void* handle;
 	void* custom;
 
+	/* clipboard capabilities - set by server */
```
Comment (bmiklautz) [inline include/freerdp/server/cliprdr.h]: @mfleisz am I mistaken or are the following values in the end the combination 
of what is supported by the server and the client?

Comment (mfleisz) [inline include/freerdp/server/cliprdr.h]: Exactly - the server first announces its capabilities (i.e. support for long format names) - if the client replies that it does not support that capability the server will revert to use use short format names in that example.

Comment (bmiklautz) [inline include/freerdp/server/cliprdr.h]: @mfleisz what do you think about updating the comment to reflect this fact? While reviewing the PR the comment confused me a little ;).

**Comment by bmiklautz** [comment]
Except my comment +1.
Nice that this can be set now \o/

**Comment by mfleisz** [comment]
@bmiklautz Done - hope this is clearer now :)

**Comment by bmiklautz** [comment]
@freerdp-bot test.

**Pull request merged**